docs/samples/Microsoft.ML.Samples/Dynamic/SDCACalibrated.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Linq;
+using Microsoft.ML.Data;
+using Microsoft.ML.Trainers;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    public class SDCACalibrated_BinaryClassificationExample
+    {
+        /// <summary>
+        /// Call <see cref="SdcaCalibratedBinaryTrainer"/> to train a logistic regression model.
+        /// Training examples are generated by calling <see cref="Samples.Utils.DatasetUtils.GenerateBinaryLabelFloatFeatureVectorSamples"/>.
+        /// The type of those examples is <see cref="SamplesUtils.DatasetUtils.FloatLabelFloatFeatureVectorSample"/>,
+        /// which contains one bool label and a float feature vector.
+        /// </summary>
```

Comment:
we prefer to use simple `//` comments in samples.

Reply:
This doc is strongly-typed. We can only delete it or use XML. Which one is more consistent to other places?

---
In reply to: [256210399](https://github.com/dotnet/machinelearning/pull/2506#discussion_r256210399) [](ancestors = 256210399)

Reply:
Somehow merge it into comments below.

---
In reply to: [256541690](https://github.com/dotnet/machinelearning/pull/2506#discussion_r256541690) [](ancestors = 256541690,256210399)